scripts/installDependencies.sh

```diff
@@ -19,34 +21,24 @@ install_node_version() {
     if [ -s "$nvm_path" ];
     then
         echo "NVM is already installed."
+        source $nvm_path
     else
         echo "Installing NVM..."
         install_nvm
-        source nvm_path
-    fi
-
-    node_command=`command -v node`
-    current_node_version=`node -v |  awk -F\. '{print $1}'`
-    # Check if correct version of node is present
-    if [[ "$node_command" ]] && [[ "$node_version" == "$current_node_version" ]];
-    then
-        echo "Correct node version is already installed."
-    else
-        echo "Node version $node_version is not installed. Installing..."
-        nvm install $node_version
-        nvm use $node_version
+        source $nvm_path
     fi
+    
+    echo "Installing Node version $node_version..."
+    nvm install $node_version
 }
 
 install_node_version
 
 echo "Node.js version $node_version has been installed."
 
-sudo apt install npm
-echo "***Installed npm***"
```

Comment:
I think you need to install `npm` for further steps

Reply:
IIRC, npm is automatically installed with nvm. Is it not? Also, installing NodeJS should also install npm, by default